lib/web/desktop.go

```diff
@@ -170,7 +176,7 @@ func (h *Handler) createDesktopConnection(
 		site:     site,
 		userAddr: r.RemoteAddr,
 	}
-	serviceConn, err := c.connectToWindowsService(sctx.cfg.RootClusterName, validServiceIDs)
+	serviceConn, err := c.connectToWindowsService(clusterName, validServiceIDs)
```

Comment:
As `clusterName` is user supplied, should we be sanitizing/validating the string first?